apps/schools/api_views/school.py

```diff
@@ -44,7 +44,7 @@ class SchoolsList(KLPListAPIView, CacheMixin):
     search_fields = ('name', 'id', 'dise_info__dise_code',)
 
     def get_queryset(self):
-        qset = School.objects.filter(status=2).distinct('id')
+        qset = School.objects.filter(id__gte=settings.STATE_STARTING_SCHOOL_ID, status=2).distinct('id')
```

Comment:
All `School.objects.filter(id__gte=settings.STATE_STARTING_SCHOOL_ID, status=2)` reflects the hack to fetch only the Schools that belong to Odisha. Should ideally be a `State` filter.